ports/raspberrypi/boards/pysquared_rp2040/mpconfigboard.mk

```diff
@@ -0,0 +1,19 @@
+USB_VID = 0x1209
+USB_PID = 0x0011
```

Comment:
Please request a pid.codes PID for this rather than using 0x0011 if it is open source. Otherwise, you can request one from Raspberry Pi.